plugin.xml

```diff
@@ -90,6 +90,14 @@
                 <framework src="ExternalAccessory.framework" />
         -->
         <framework src="src/ios/libs/BRLMPrinterKit.framework" custom="true"/>
+        <podspec>
+            <config>
+                <source url="https://github.com/CocoaPods/Specs.git"/>
```

Comment:
Thanks for the PR @TristanGodal  .
Are we sure this link is required?

Reply:
On my side yes, actually I was not able to run the plugin anymore cause of the deprecation of "framework" tag, so I did it by this way, console asked me to use podspec instead.